app/stores/SidebarStore.js

```diff
@@ -12,24 +12,19 @@ export default class SidebarStore extends Store {
     SETTINGS: ROUTES.SETTINGS.ROOT,
   };
 
-  INITIAL_HIDE_SUB_MENU_DELAY = 4000;
-  ACTION_HIDE_SUB_MENU_DELAY = 1000;
+  DEFAULT_WINDOW_WIDTH = 1150;
 
   @observable activeSidebarCategory: string = this.CATEGORIES.WALLETS;
-  @observable isShowingSubMenus: boolean = true;
-
-  _hideSubMenuTimeout = null;
+  @observable isShowingSubMenus: bool = false;
 
   setup() {
     const actions = this.actions.sidebar;
     actions.toggleSubMenus.listen(this._toggleSubMenus);
     actions.activateSidebarCategory.listen(this._onActivateSidebarCategory);
     actions.walletSelected.listen(this._onWalletSelected);
-    this.actions.networkStatus.isSyncedAndReady.listen(() => {
-      this._hideSubMenusAfterDelay(this.INITIAL_HIDE_SUB_MENU_DELAY);
-    });
     this.registerReactions([
       this._syncSidebarRouteWithRouter,
+      this._setInitialSubMenusState,
```

Comment:
You should not register this as reaction since `_setInitialSubMenusState` should only be run once and does not depend on any state from what i see?